src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { getTickerList } from './utils/get-ticker-list'
+import { getDataForTickers } from './utils/scrape-data-for-tickers'
+
+const main = async () => {
+  const tickerList = getTickerList()
+
+  console.log('Input data: ', tickerList)
```

Comment:
Wrap this with is DEBUG so we don't log in production

Reply:
true, I could also change them to "console.debug"

Reply:
I changed it to a logger from the "winston" library. This allows you to suppress logs if you want to do that, but the main thing I like about this is that I set it up to write the logs to a file too (useful for when it's running on a server as a cronjob and you come later to check if everything went ok or errored)